cosmos-gui/views/dashboard.jade

```diff
@@ -1,6 +1,31 @@
+//
```

Comment:
Well, when learning how to create web application in Node.js I found a lot of blogs and Internet places talking about Jade and suggesting it was the better option for this purpose... So I followed them :) I found it easy to use, btw.